utils.py

```diff
@@ -72,10 +72,10 @@ def symbol_address(binary_path: str, symbol: str) -> Optional[int]:
 
     symbol_bytes = symbol.encode()
 
-    def exact_match(name, addr, payload, _):
+    def exact_match(name, addr, length, payload):
         nonlocal result
         if name == symbol_bytes:
-            result = addr
+            result = (addr, length)
```

Comment:
done. Take a look. Not sure if you like fields names. I used an Offset instead of Addr because it reflects the semantics of the function clients. Can change it to Addr if you want.